src/ReturnController.php

```diff
@@ -306,6 +306,14 @@ public function rest_api_adyen_redirect( WP_REST_Request $request ) {
 			$payment->set_status( $status );
 		}
 
+		$payment->add_note(
+			\sprintf(
+				/* translators: %s: Adyen payment result code. */
+				\__( 'Result code `%s` was received via the Adyen web drop-in, awaiting the webhook to update payment status.', 'pronamic_ideal' ),
```

Comment:
Is 'Adyen web drop-in' clear to users?

```suggestion
				\__( 'Received preliminary result code `%s` by client. Awaiting webhook to update payment status.', 'pronamic_ideal' ),
```

Reply:
The note is mainly for administrators, they probably consulted https://docs.adyen.com/online-payments/web-drop-in for the configuration? The term 'client' may not always be clear, do we mean the customer? And maybe 'preliminary' doesn't quite cover it either? Without abuse it will just be the correct `resultCode`. However, we cannot process this for security reasons.